mirrord-layer/src/lib.rs

```diff
@@ -42,6 +43,7 @@ mod pod_api;
 mod socket;
 mod tcp;
 mod tcp_mirror;
+mod tcp_steal;
```

Comment:
Same module suggestion I've given. It would be nicer to have a `tcp` module, and put these `tcp_` under it (also removing the `tcp_` part from names).